wasp/apps/morse.py

```diff
@@ -0,0 +1,148 @@
+# SPDX-License-Identifier: LGPL-3.0-or-later
+# Copyright (C) 2021 Francesco Gazzetta
+
+"""Morse translator and notepad
+~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
This line is triggering CI failures (the underlining should line up with the d in pad)

Reply:
Fixed!